phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/default/main.less

```diff
@@ -101,16 +101,12 @@
 /*******************************************************/
 .leftPane {
   -fx-background-color: @background-color;
-  -fx-min-height: 10em;
-  -fx-min-width: 15em;
-  -fx-pref-width: 15em;
   -fx-padding: 1em;
 
   .leftPaneScrollbar {
     -fx-border-width: 0;
     -fx-background-color: @background-color;
-    -fx-min-width: 15em;
-    -fx-padding: 0.8em 1em 0.8em 1em;
+    -fx-padding: 0.8em 0 0.8em 0;
```

Comment:
Are you sure?
The unity theme doesn't define a `-fx-padding` for the `.leftPaneScrollbar`, so it should use the default one.